### PR TITLE
Feature: lesson01

### PR DESCRIPTION
**Pull request by haru-programming**
# [Issue No.1](https://github.com/kenmori/handsonFrontend/blob/master/work/markup/1.md#1%E3%81%93%E3%81%AEdom%E3%82%92html%E5%86%85%E3%81%AEul%E3%81%AE%E4%B8%AD%E3%81%AB%E5%B7%AE%E3%81%97%E8%BE%BC%E3%82%93%E3%81%A7%E3%81%8F%E3%81%A0%E3%81%95%E3%81%84)
## Concerns and areas to focus on
I think I chose the best method by referring to the following, is it ok?<br>
- I have selected 'appendChild'.I referred to the [StackOverflow](https://stackoverflow.com/questions/16126960/what-is-the-difference-between-appendchild-insertadjacenthtml-and-innerhtml).<br>
> The appendChild methods adds an element to the DOM.<br>
> The innerHTML property and insertAdjacentHTML method takes a string instead of an element, so they have to parse the string and create elements from it, before they can be put into the DOM.<br>

- I have selected 'textContent'.I referred to the [StackOverflow](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent).<br>
> Moreover, since innerText takes CSS styles into account, reading the value of innerText triggers a reflow to ensure up-to-date computed styles. (Reflows can be computationally expensive, and thus should be avoided when possible.)

Thank you for the review.

### CODE REVIEW

File: lesson01/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const ul = document.getElementById('js-lists');
+const li = document.createElement('li');
+
+ul.appendChild(li);
+li.textContent = 'これです';
```
Comment (kenmori) [inline lesson01/index.js:7]: IMO
```suggestion
li.textContent = 'これです';
ul.appendChild(li);

```

**Comment by kenmori** [comment]
> I think I chose the best method by referring to the following, is it ok?

good. I agree.

**Comment by kenmori** [comment]
nits

1.
> feat: lesson01/add li to ul #1

PR title is created with commit message by default. Change it appropriately.

File: lesson01/index.js (hunk 1 above)
Comment (haru-programming) [reply lesson01/index.js]: Thank you for the review!
I thought this order is more natural.
1. create li
2. change the text in li
3. insert li to ul

I thought it was important, so I fixed it.thank you.
b80b295

**Comment by haru-programming** [comment]
> 1. PR title is created with commit message by default. Change it appropriately.

thank you for the review.
I fixed PR title  to 'Feature: lesson01'.

----------
> 2. commit message(`feat: lesson01/add li to ul`) is no need `lesson01/`.

The commit message was the part that bothered me.
Thank you very much.
I'll use it in the next time immediately!

**Comment by kenmori** [comment]
Approve

**Pull request merged**